docs/user/update_states.rst

```diff
@@ -94,6 +102,50 @@ Bodhi will close associated bugs, and will send out update notices to the approp
 addresses.
 
 
+.. _sidetag-active:
+
+Side_tag_active
+===============
+
+An update can be created as a side tag. This corresponds to the request for a Koji side tag, which is
+a build target used to collect and iterate builds temporarily. This allows builds to be iterated without
+interfering with content in tags that ship to consumers. Once the builds are complete and correct, the
+side tag can then be merged into an existing tag.
+
+This state represents content for a side tag that is still modifiable.
```

Comment:
This state represents a side-tag whose content is still modifiable?

The ``represents content`` is what bugged me a little basically :)